examples/kafka_filter/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye-slim as builder
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    curl \
+    ca-certificates \ 
+    cmake \
+    pkg-config \
+    libsasl2-dev \
+    flex \
+    bison
+WORKDIR /build/
+ADD . /source
+RUN cmake -DFLB_DEV=On \
+          -DFLB_IN_KAFKA=On \
+          -DFLB_OUT_KAFKA=On \
+          /source
+RUN make -j "$(getconf _NPROCESSORS_ONLN)"
```

Comment:
We probably should update the main dockerfiles to just build the input plugin and use that. In fact the default in the CMakeLists would handle it I think as this is built for each PR: `ghcr.io/fluent/fluent-bit/pr-X`. Then you just want to mount the config in as part of the compose stack.